### DISCUSSION TITLE
Sns notifications to webhook

### DISCUSSION THREAD

**Discussion by PLPRASU** [StackStorm/st2#5646] [Q&A]
can i subscribe webhook url in sns subscription is this possible, if possible how i confirm sns protocol type is https

**Comment by nzlosh** [comment]
It looks like it's possible with the AWS pack https://github.com/StackStorm-Exchange/stackstorm-aws

**Reply by PLPRASU** [reply]
I am trying to send AWS SNS notifications to stackstorm webhooks but unable to confirm subscription as event is not getting logged on the webhook.
In the nginx logs, I can see that subscribe confirm event hit the nginx but cannot find the payload of the sns event on webhook to fetch confirmation token or subscribbe url ?
Is there a way to get it ?

**Reply by vasanthdigitalxc** [reply]
I am also trying the same thing.
